src/displayapp/widgets/Counter.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <lvgl/lvgl.h>
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Widgets {
+      class Counter {
+      public:
+        Counter(int min, int max);
+
+        void Create();
+        static void upBtnEventHandler(lv_obj_t* obj, lv_event_t event);
+        static void downBtnEventHandler(lv_obj_t* obj, lv_event_t event);
+        void Increment();
+        void Decrement();
+        void SetValue(int newValue);
+
+        int GetValue() const {
```

Comment:
I'd move the body of the function into the cpp file to prevent multiple definition errors (just best practice, for our smallish project with just one binary it should be no problem, but better be safe)

Reply:
I think it needs to be in the header file for it to get inlined.

Reply:
I didn't know that, thanks!

Stackoverflow provided me an additional interesting read https://stackoverflow.com/questions/453372/writing-function-definition-in-header-files-in-c